README.md

```diff
@@ -213,3 +279,16 @@ being maintained and conflicting with [wartremover](https://github.com/wartremov
 ### SBT Tasks
 
 No special tasks are enabled for this plugin.
+
+# Releasing of the `sbt-cake` Plugin to the Sonatype Repository
+
+In order to release version `X.Y.Z` of the `sbt-cake` plugin to the `net.cakesolutions` Sonatype repository, perform the
+following actions:
+```text
+git tag -s vX.Y.Z
+sbt publishSigned sonatypeRelease
+git push --tags
+```
+
+In order for releasing to occur, individuals should have the Cake Solutions Sonatype PGP keys setup in their local
```

Comment:
I don't agree that devs should be publishing releases, i would like only Jenkins to do the releasing.  What are thoughts regarding this?  Is this something that can be integrated into CI easily?